types/job_info.go

```diff
@@ -0,0 +1,35 @@
+package types
+
+import (
+	"encoding/json"
+	"time"
+)
+
+type JobInfo struct {
+	ID        string
+	Project   string
+	StartedAt time.Time
+	Output    string
+	Log       string
+	State     string
+}
```

Comment:
I see many duplicated fields between this and `Job`. Can we merge this into `Job`? It seems redundant.

Reply:
Bump.

Reply:
Done.